### ISSUE TITLE
Failed to load resource

### ISSUE THREAD

**Issue by ErnestClose** [meteor-utilities/avatar#34]
Getting failed to load resource: bad request for packages/avatar/default.png
I can see default.png in packages/utilities_avatar/default.png and there is no packages/avatar

**Comment by timothyarmes** [comment]
Sounds like you're not using the latest version.

**Comment by ErnestClose** [comment]
I don't know, I just installed it yesterday with meteor add utilities:avatar.

**Comment by timothyarmes** [comment]
I believe that this is now fixed.

**Comment by ErnestClose** [comment]
Updated to latest version, now get this:
GET http://i0.wp.com/localhost/packages/utilities_avatar/default.png 404 (Not Found)

**Comment by timothyarmes** [comment]
Can you check inside your .meteor build folder to see what the path should be? On my system that what I get.

**Comment by ErnestClose** [comment]
I get it here: .meteor\local\build\programs\server\assets\packages\utilities_avatar
I like the initials that display if there is no gravatar, but still get the above 404 error every time the page loads. If it shows either the gravatar photo or the initials if gravatar is not available, then why/when does it show the default.png?

**Comment by timothyarmes** [comment]
The `fallback` type option determines whether the package will show initials or the default avatar when there's no other image available.

Make sure you have the version 0.9.2. You should be able to access the default image by going to http://localhost:3000/packages/utilities_avatar/default.png (when you running locally, obviously). It's certainly working for me.

**Comment by ErnestClose** [comment]
Yes I can access it with that path, because that is where it is, but the error shows this path GET http://i0.wp.com/localhost/packages/utilities_avatar/default.png 404 (Not Found) and that is not on localhost but i0.wp.com 
I have the latest version.

**Comment by rcarter** [comment]
+1 getting the same thing here... GET http://i0.wp.com/localhost/packages/utilities_avatar/default.png 404 (Not Found)

**Comment by gdw2** [comment]
The README says:

> Note that Gravatar's default option requires a publicly accessible URL, so it won't work when your app is running on localhost and you're using either the included package default image or a custom defaultImageUrl that is a relative path. It will work fine once deployed though.

Could this be your problem?

**Comment by romant** [comment]
+1

**Comment by daveeel** [comment]
Yes I could confirm that the problem doesn't exist on deployment. However it's nice to remove the error on localhost dev.

**Comment by guillim** [comment]
+1
